README.md

```diff
@@ -284,24 +284,28 @@ radio buttons so only the valid ones are displayed to users.
 See [the SpringRoll Application Class docs](https://github.com/SpringRoll/SpringRoll/tree/develop/src#responding-to-the-container) for more information on
 the request format and how game developers provide those values.
 
-### ControlsPlugin
+### Controls
+There are two plugins associated with in-game controls: `ControlSensitivityPlugin` and `KeyboardMapPlugin`.
+The Control Sensitivity Plugin allows the user to determine the sensitivity of custom pointers in game. This plugin expects an HTML Input
+The Keyboard Map Plugin allows users to re-map the keys/controls used in-game to something they are more comfortable with.
+
 ```javascript
-import { ControlsPlugin, Container } from 'springroll-container';
+import { ControlSensitivityPlugin, KeyboardMapPlugin, Container } from 'springroll-container';
 
 const container = new springroll.Container('#game', {
   plugins: [
-    //ControlsPlugin also accepts an [optional] initial value for its control sensitivity
-    new ControlsPlugin({
-      //Expects an HTML Input Element of type="range"
-      sensitivitySliders: '#sensitivity-slider-selector',
+    //ControlSensitivityPlugin expects an input of type=range for it's input.
+    new ControlSensitivityPlugin('#sensitivity-slider-selector', {
       defaultSensitivity: 0.5, //control sensitivity goes from 0.0 to 1.0. Default = 0.5
-      keyContainers: '#key-container', // container element that will contain the mappable key buttons.
     }),
+    //The KeyboardMapPlugin expects a div or similar container element as it's selector. It will automatically build out the
+    //buttons to use for re-mapping controls based on what the application returns.
+    new KeyboardMapPlugin('#key-container')
```

Comment:
We should probably document how the automatically built-out DOM elements would look. That would help devs style those elements if need be.